<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StyleEditor</class>
 <widget class="QMainWindow" name="StyleEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>608</width>
    <height>517</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Style Editor - TikZiT</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QComboBox" name="currentCategory">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>181</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelName">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>50</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Name</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelCategory">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>80</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Category</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelFillColor">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>110</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Fill Color</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelDrawColor">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>140</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Draw Color</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="name">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>50</y>
      <width>301</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QComboBox" name="category">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>80</y>
      <width>301</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="editable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="fillColor">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>110</y>
      <width>31</width>
      <height>18</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="drawColor">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>140</y>
      <width>31</width>
      <height>18</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="hasTikzitFillColor">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>110</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <italic>false</italic>
     </font>
    </property>
    <property name="text">
     <string>in TikZiT:</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="hasTikzitDrawColor">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>140</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <italic>false</italic>
     </font>
    </property>
    <property name="text">
     <string>in TikZiT:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="tikzitFillColor">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>110</y>
      <width>31</width>
      <height>18</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="tikzitDrawColor">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>140</y>
      <width>31</width>
      <height>18</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="labelShape">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>170</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Shape</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QComboBox" name="shape">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>170</y>
      <width>231</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="editable">
     <bool>true</bool>
    </property>
    <item>
     <property name="text">
      <string/>
     </property>
    </item>
    <item>
     <property name="text">
      <string>circle</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>rectangle</string>
     </property>
    </item>
   </widget>
   <widget class="QCheckBox" name="hasTikzitShape">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>200</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <italic>false</italic>
     </font>
    </property>
    <property name="text">
     <string>in TikZiT:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="tikzitShape">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>200</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <item>
     <property name="text">
      <string/>
     </property>
    </item>
    <item>
     <property name="text">
      <string>circle</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>rectangle</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="labelArrowhead">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>230</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Arrowhead</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QComboBox" name="leftArrow">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>230</y>
      <width>41</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="editable">
     <bool>false</bool>
    </property>
    <item>
     <property name="text">
      <string/>
     </property>
    </item>
    <item>
     <property name="text">
      <string>&lt;</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>|</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="labelDash">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>230</y>
      <width>16</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QComboBox" name="rightArrow">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>230</y>
      <width>41</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="editable">
     <bool>false</bool>
    </property>
    <item>
     <property name="text">
      <string/>
     </property>
    </item>
    <item>
     <property name="text">
      <string>&gt;</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>|</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="labelProperties">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>270</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Properties</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QToolButton" name="addProperty">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>460</y>
      <width>23</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QToolButton" name="addAtom">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>460</y>
      <width>23</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>+a</string>
    </property>
   </widget>
   <widget class="QToolButton" name="removeProperty">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>460</y>
      <width>23</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QToolButton" name="propertyUp">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>460</y>
      <width>23</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>^</string>
    </property>
   </widget>
   <widget class="QToolButton" name="propertyDown">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>460</y>
      <width>23</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>v</string>
    </property>
   </widget>
   <widget class="QListView" name="styleListView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>181</width>
      <height>251</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
      <italic>true</italic>
     </font>
    </property>
   </widget>
   <widget class="QListView" name="edgeStyleListView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>330</y>
      <width>181</width>
      <height>151</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
      <italic>true</italic>
     </font>
    </property>
   </widget>
   <widget class="QToolButton" name="addStyle">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>23</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QToolButton" name="removeStyle">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>300</y>
      <width>23</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QToolButton" name="styleUp">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>300</y>
      <width>23</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>^</string>
    </property>
   </widget>
   <widget class="QToolButton" name="styleDown">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>300</y>
      <width>23</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>v</string>
    </property>
   </widget>
   <widget class="QToolButton" name="removeEdgeStyle">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>490</y>
      <width>23</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QToolButton" name="addEdgeStyle">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>490</y>
      <width>23</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QToolButton" name="edgeStyleUp">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>490</y>
      <width>23</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>^</string>
    </property>
   </widget>
   <widget class="QToolButton" name="edgeStyleDown">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>490</y>
      <width>23</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>v</string>
    </property>
   </widget>
   <widget class="QPushButton" name="save">
    <property name="geometry">
     <rect>
      <x>479</x>
      <y>490</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Save and Close</string>
    </property>
   </widget>
   <widget class="QTreeView" name="properties">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>270</y>
      <width>301</width>
      <height>181</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="rootIsDecorated">
     <bool>false</bool>
    </property>
    <property name="uniformRowHeights">
     <bool>true</bool>
    </property>
    <property name="headerHidden">
     <bool>false</bool>
    </property>
    <attribute name="headerVisible">
     <bool>true</bool>
    </attribute>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
